crypto/bls12381/key_bls12381.go

```diff
@@ -18,6 +20,21 @@ const (
 	Enabled = true
 )
 
+var (
+	// ErrDeserialization is returned when deserialization fails.
+	ErrDeserialization = errors.New("bls12381: deserialization error")
+
+	dstMinSig = []byte("BLS_SIG_BLS12381G1_XMD:SHA-256_SSWU_RO_NUL_")
+)
+
+// For minimal-pubkey-size operations.
```

Comment:
We need to change this to minimal-signature-size operations because we care about signature size more!

Reply:
Can you open an issue for that?

Reply:
https://github.com/cometbft/cometbft/issues/3624